src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx

```diff
@@ -24,6 +24,31 @@ export default {
       control: { type: "text" },
       defaultValue: "Se din profil",
     },
+    notifications: {
+      control: { type: "object" },
+      defaultValue: [
```

Comment:
Lovely that you adopted this. I'm really enjoying argTypes and default values too in my stories 😼